### ISSUE TITLE
Dialog: Project creation

### ISSUE THREAD

**Issue by clstaudt** [tuttle-dev/tuttle#73]

**Comment by clstaudt** [comment]
On the right track here, nice look & feel.

But when I want to create a new client and contract, I cannot add the required information (e..g contract terms).


![Image](https://user-images.githubusercontent.com/875194/204139226-c014877d-37b8-4280-8ad6-015cfd047791.png)

**Comment by vlad-ed-git** [comment]
Yes, this is by design to keep the pop ups are light weight and the action
smooth (so you can quickly define a project with only the necessary
information. We can add to them ofcourse, currently it's only the title
that is needed for the client and the contract) . The idea being you can
define other information by editing either the client or the contract
later. Else the UI would have too many fields (the contract for instance
has unit, rate, vate rate, currency, etc... )

**Comment by clstaudt** [comment]
@vlad-ed-git  I have thought about this and I think I'm not in favor of allowing the creation of incomplete model objects. Extra code would need to be added to check whether a model object has the necessary fields set, and handle errors if it is incomplete.

**Comment by vlad-ed-git** [comment]
okay, editing this to include other fields.

**Comment by clstaudt** [comment]
@vlad-ed-git  nicely solved, UI ready for user testing
